frontend/components/product-list/sections/FilterableProductsSection/ProductList.tsx

```diff
@@ -259,12 +263,12 @@ export function ProductListItem({ product }: ProductListItemProps) {
                      </Button>
                   </LinkOverlay>
                   <Text color="gray.500" fontSize="14px">
-                     {product.inventory_quantity > 0 &&
-                     product.inventory_quantity < 10
+                     {product.quantity_available > 0 &&
+                     product.quantity_available < 10
                         ? `
-               Only ${product.inventory_quantity} left in stock
+               Only ${product.quantity_available} left in stock
                `
-                        : `${product.inventory_quantity} in stock`}
+                        : `${product.quantity_available} in stock`}
```

Comment:
I don't know if this was explained, but `quantity_available` is clamped to `[0,10]` i.e. if there are actually 55, `quantity_available` will be 10. Given that, I might just exclude this text if the number is 10 or more.

Thoughts?

Reply:
The refactor is only due to the fact that the new index has `quantity_available` and doesn't have `inventory_quantity`.
Anyway your suggestion to hide the quantity when ≥ 10 makes sense to me 👍 

@sterlinghirsh what do you think?

Reply:
Also there's https://github.com/iFixit/react-commerce/issues/199 tracking this, let's decide on this right now so we can close it.

cc @erinemay @Annika29 @lithobraking

Reply:
My vote is to only show product quantity when it's low (1-10). If you look at that issue (https://github.com/iFixit/react-commerce/issues/199), we get huge counts displayed (`3073 in stock`), and we get `0 in stock` even when we already have a "Sold Out" tag on the card. I don't know how often our inventory counts get into a negative state, or if it's just a result of the current, imperfect Algolia index, but we can also get negative stock quantities displayed as well (https://github.com/iFixit/react-commerce/issues/193).

Reply:
@erinemay I've already fixed negative quantities on another branch, so I think let's merge this as is and apply the polishing with the next PR ok?